contracts/token/MintableToken.sol

```diff
@@ -0,0 +1,46 @@
+pragma solidity ^0.4.8;
+
+
+import './StandardToken.sol';
+import '../ownership/Ownable.sol';
+
+
+
+/**
+ * Mintable token
+ *
+ * Simple ERC20 Token example, with mintable token creation
+ * Issue:
+ * https://github.com/OpenZeppelin/zeppelin-solidity/issues/120
+ * Based on code by TokenMarketNet:
+ * https://github.com/TokenMarketNet/ico/blob/master/contracts/MintableToken.sol
+ */
+contract MintableToken is StandardToken, Ownable {
+
+  bool public mintingFinished = false;
+  uint public totalSupply = 0;
+
+  modifier canMint() {
+    if(mintingFinished) throw;
+    _;
+  }
+
+  function mint(address _to, uint _amount) onlyOwner canMint returns (bool) {
+    totalSupply = safeAdd(totalSupply, _amount);
+    balances[_to] = safeAdd(balances[_to], _amount);
+    Transfer(0, _to, _amount);
```

Comment:
I think you're right, my initial intention was to limit the amount of additional events to make searching for them easier but it's probable different enough from Transfer that it deserves its own event.